src/app/cards/munchkin/cards/treasures/disposable-item/edit/disposable-item-edit.component.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  <app-editable-string [(string)]="card.name" [label]="'name'"></app-editable-string>
+
+  <app-editable-rich-text [(text)]="card.description" [label]="'description'" [multiline]="'true'">
+  </app-editable-rich-text>
+  <app-editable-number [(number)]="card.price" min="0" max="2000" step="100" [label]="'price'">
```

Comment:
See [here](https://github.com/NastyaKolomoets/card-editor/pull/45#discussion_r484550732).

Reply:
Will create ticket for this :)